integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java

```diff
@@ -119,9 +119,9 @@ public static void deleteClusterCustomResourceAndVerify(String clusterName, Stri
     Cluster.deleteClusterCustomResource(clusterName, namespace);
 
     testUntil(
+        withLongRetryPolicy,
         clusterDoesNotExist(clusterName, CLUSTER_VERSION, namespace),
-        getLogger(),
-        "cluster {0} to be created in namespace {1}",
+        getLogger(), "cluster {0} to be created in namespace {1}",
```

Comment:
isn't it that the method waits for the cluster to be deleted?

Reply:
No Sankar, it doesn't wait:

  public static boolean doesClusterExist(String clusterResName, String clusterVersion, String namespace) {

    Object clusterObject = null;
    try {
      clusterObject
          = customObjectsApi.getNamespacedCustomObject(
          "weblogic.oracle", clusterVersion, namespace, "clusters", clusterResName);
    } catch (ApiException apex) {
      getLogger().info(apex.getMessage());
    }
    boolean cluster = (clusterObject != null);
    getLogger().info("Cluster Object exists : " + cluster);
    return cluster;
  }

Reply:
Sorry, Sankar. I misunderstood you, Yes. the log msg is wrong. I just changed it